proposal.rst

```diff
@@ -271,7 +312,34 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+Community Bonding Period (May 4 - May 31)
+    * Interact with mentors, introduce myself to the community.
+    * Get familiar with Xapian’s development style. Understand expected coding, documentation, and testing standards set by Xapian.
+    * Start working on some issues to familiarise with the codebase. (Not necessarily related to the project)
+    * Discuss and prepare a list of various MIME types that need to be worked on.
+    * Read documentation relating to the required libraries.
+
+First Coding Period (June 1 - June 28)
+    * Implement first handler, update build system, and add new worker to omindex. (7 days)
+    * Look for issues, take feedback, improve performance. (6 days)
+    * Document the code. (4 days)
+    * Finalise the PR and merge. (2 days)
+
+Second Coding Period (July 6 - July 27)
+    For two types:
```

Comment:
What types are you proposing? It's fine to suggest something here but change your mind after later discussion. But having something concrete laid out in your proposal allows you to think about the specifics of the work needed. "Add support for a new type" is very abstract. Are you going to add support for a single file format using a specialist library (perhaps from the Document Liberation Project)? Or are you going to find another library that could support several different file formats at once (which might take more work)?

Reply:
I am proposing work on OpenDocument formats only. Updated the proposal.